include/internal/common.h

```diff
@@ -38,7 +38,7 @@
 # endif
 
 # ifdef NDEBUG
-#  define ossl_assert(x) ((x) != 0)
+#  define ossl_assert(x) likely((x) != 0)
```

Comment:
Why is this global change hidden in here? It should be raised as a separate pull request imo.

Reply:
#22516